src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Image.cs

```diff
@@ -434,6 +435,40 @@ void Image_ImageOpened(object sender, RoutedEventArgs e)
 			Assert.AreEqual("Image_ImageOpened", logs[1]);
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Exif_Rotated_MsAppx()
+		{
+			await When_Exif_Rotated_Common(new Uri("ms-appx:///Assets/testimage_exif_rotated.jpg"));
+		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Exif_Rotated_MsAppData()
+		{
+			var file = await StorageFile.GetFileFromApplicationUriAsync(new Uri("ms-appx:///Assets/testimage_exif_rotated.jpg"));
+			var fileName = $"{Guid.NewGuid()}.jpg";
+			await file.CopyAsync(ApplicationData.Current.LocalFolder, fileName);
+			var uri = new Uri($"ms-appdata:///Local/{fileName}");
+			await When_Exif_Rotated_Common(uri);
+		}
+
+		private async Task When_Exif_Rotated_Common(Uri uri)
+		{
+			var image = new Image();
+			var bitmapImage = new BitmapImage(uri);
+			var imageOpened = false;
+			image.ImageOpened += (_, _) => imageOpened = true;
+			image.Source = bitmapImage;
+			WindowHelper.WindowContent = image;
+			await WindowHelper.WaitForLoaded(image);
+			await WindowHelper.WaitFor(() => imageOpened);
+			var screenshot = await TakeScreenshot(image);
+			var topCenterActual = screenshot.GetPixel(screenshot.Width / 2, 5);
```

Comment:
This is likely to fail on wasm (you can gate the feature on ApiInformation for rendertargetbitmap).

Reply:
It may be the issue of tests being skipped because of invalid buckets... I'll have to backport parts of that breaking changes PR to ensure that all runtime tests are run properly.